src/frontend/components/UploadManager/index.tsx

```diff
@@ -58,6 +59,7 @@ export const UploadManager = ({ children }: React.PropsWithChildren<{}>) => {
             file!.name,
             file!.type,
           );
+          await getResource(objectType, objectId);
```

Comment:
As the upload manager is to be used in the site as well as the LTI app, we will have to remove this call so it can work with the site.

The purpose is to update the resource in the store right? Can this wait until we set the state to `UploadManagerStatus.UPLOADING` below (line 86) so it can be done in a Marsha LTI component?

Reply:
It can be moved to the `LTIUploadHandlers` I think

Reply:
Ok thanks! I can take care of that in my own PR then, this stuff is not in prod yet anyway 👍